### PR TITLE
Feature Add Api

### PR DESCRIPTION
**Pull request by CarlosZ96**
Connecting to the ninjas API to get the emoji data, bind the character property to get the image and display it next to the tittle

### CODE REVIEW

File: src/components/quote.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+async function emoji(callback) {
+  const name = 'slightly%20smiling%20face';
+  const apiKey = '2VyJFEAwAfCwJeQRTJpUKw==qoL1KftFFfAoQ4th';
+
+  try {
```
Comment (Lizdev-05) [inline src/components/quote.jsx]: - [ ] Please ensure that you create a `<Quotes>` component and in this component make sure the `useEffect` hook has been used correctly
    1. There should be a `loading` state for the API call
    2. There should be an `error` state for the API call.

File: src/App.jsx [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+      <div className="Title-container">
+        <h1 className="title">Math Magicians</h1>
+        {quoteData && (
+          <div className="quote">
+            <span className="emoji">{quoteData.character}</span>
+          </div>
+        )}
       </div>
       <Calculatorb />
     </div>
```
Comment (Lizdev-05) [inline src/App.jsx]: - [ ] It would be better if you ensure that the `<Quotes>` component is imported into this `App` component and rendered. :blush:

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @CarlosZ96 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — COMMENTED** [review]
Hi @CarlosZ96 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Expected states are correctly implemented for the data fetching process

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @CarlosZ96 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Expected states are correctly implemented for the data fetching process

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**